test-runner-poc/lib/runner.js

```diff
@@ -27,47 +30,77 @@ function addFilter(filter) {
 	}
 }
 
-before(function() {
+function environmentSetup(done) {
```

Comment:
It looks like the formatting is a little off in this whole file. Can we set the indentation to 2 spaces? The area around the switch statement looks particularly over-indented.

Reply:
fixed indentation!